fem/fe.hpp

```diff
@@ -731,6 +732,15 @@ class NodalFiniteElement : public ScalarFiniteElement
    virtual void ProjectDiv(const FiniteElement &fe,
                            ElementTransformation &Trans,
                            DenseMatrix &div) const;
+
+   /** @brief Get an Array<int> that maps lexicographically ordered indices to
```

Comment:
I think we should define exactly what is meant by "lexicographically ordered".  Without knowing the lexicon and the rules we use to order it this array may not be useful.

Reply:
How does the updated comment look to you?

Reply:
Looks perfect. Thanks!